src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -78,6 +90,8 @@ const getMessageFromBraze = async (
     });
 
     return new Promise((resolve) => {
+
+        appboyTiming.start();
```

Comment:
It feels to me like the `appboyTiming` should start when we initialize the SDK and stop/record when (if) we get a message back. What do you think? That would allow us to scope the `const appboyTiming = initPerf('braze-appboy');` more closely to the timing calls, instead of having it be global at the top. So init and start just before `appboy.initialize` and then stop/record when the promise we're in here resolves. What do you think?

Reply:
Hey Tom, thanks for looking over this! Sounds good to me - I've updated the PR now, let me know what you think.